core/src/main/java/io/aiven/klaw/repository/MessageSchemaRepo.java

```diff
@@ -26,10 +26,10 @@ List<MessageSchema> findAllByTenantIdAndTopicnameAndSchemaversionAndEnvironment(
       int tenantId, String topicName, String schemaVersion, String environmentId);
 
   @Query(
-      value = "select count(*) from kwavroschemas where env = :envId and tenantid = :tenantId",
+      value =
+          "select exists(select 1 from kwavroschemas where env = :envId and tenantid = :tenantId)",
       nativeQuery = true)
-  List<Object[]> findAllSchemaCountForEnv(
-      @Param("envId") String envId, @Param("tenantId") Integer tenantId);
+  boolean existsSchemaForEnv(@Param("envId") String envId, @Param("tenantId") Integer tenantId);
```

Comment:
Exists statements can be written without the need to add a native query e.g. 
https://github.com/Aiven-Open/klaw/blob/d39f8e2fa1b04dd6ebc8d23ae4f3dc1846281d23/core/src/main/java/io/aiven/klaw/repository/TopicRequestsRepo.java#L18

Which i think would be better then us maintaining our native queries. (unless their is a perf improvement?)